apps/webhooks.md

```diff
@@ -63,6 +65,65 @@ A notification would look like this:
 
 Depending on server traffic, notifications on events might take a few seconds.
 
+### Verify callback requests
+
+Each callback ePages sends to registered webhooks contains a `X-Epages-Hmac-Sha256` header with a signature that can be used to verify that the request is really coming from ePages.
+This signature is calculated by
+
+**signature = encode_Base64( HMAC_SHA256(secret, message) )**
+
+where message is the request body of the callback request and secret is the OAuth2 Client Secret of the app which has registered the webhook. (It is the one, the developer gets when they create an App on our developer platform).
```

Comment:
as secret comes first and message comes second, I'd put it like this:

where `secret` is the OAuth2 Client Secret of the app which has registered the webhook (the one, the developer receives when creating an app on the developer portal) and `message` is the request body of the callback request.